src/tribler/core/components/restapi/restapi_component.py

```diff
@@ -51,12 +52,15 @@ def maybe_add(self, path: str, endpoint_cls: Type[RESTEndpoint], *args, **kwargs
         """ Add the corresponding endpoint to the path in case there are no `NoneComponent`
         in *args or **kwargs
         """
+        self.logger.info(f'Adding: "{path}"...')
         arguments_chain = chain(args, kwargs.values())
-        if any(isinstance(arg, NoneComponent) for arg in arguments_chain):
-            self.logger.info("Endpoint %s can't load component, skipping init", path)
+        need_to_skip = any(isinstance(arg, NoneComponent) for arg in arguments_chain)
+        if need_to_skip and not self.swagger_mode:
+            self.logger.warning("Skipped")
             return
 
         self.root_endpoint.add_endpoint(path, endpoint_cls(*args, **kwargs))
+        self.logger.info("OK")
```

Comment:
**nitpick:** A single `"OK"` log line may be uninformative. How about changing it to something like "Endpoint XXX added" or removing it entirely?

Reply:
From my POV lines like "Endpoint XXX added" just after "Adding XXX..." lead only to the pollution of the logs.